staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -113,6 +113,14 @@ var (
 		},
 		[]string{"resource"},
 	)
+	decodeErrorCounts = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "storage_decode_errors",
+			Help:           "Number of stored object decode errors split by object type.",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"type"},
```

Comment:
`type` in an error metric is quite confusing since it might imply that it stores the type of the error, `resource` would be better suited and more inline with the existing metrics.

Reply:
will update. thanks